feat(curriculum): add Caesar cipher workshop (#60198)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/workshop-caesar-cipher/6810869355fa2468f9ebaf9c.md

---
id: 6810869355fa2468f9ebaf9c
title: Step 9
challengeType: 20
dashedName: step-9
---

# --description--

> Now print `encrypted_text` on the terminal and see the output.

Reviewer: ```suggestion
Now print `encrypted_text` to see the output in the terminal.
```
